ycms/cms/models/ward.py

```diff
@@ -60,7 +70,12 @@ def patients(self):
         :return: patients in the ward
         :rtype: list [ ~ycms.cms.models.patient.Patient ]
         """
-        return Patient.objects.filter(bedassignment__bed__room__ward=self).distinct()
+        # return Patient.objects.filter(medical_records__bed_assignment__bed__room__ward=self).distinct()
+        # filter patients whos current stay is in this room, namely no discharge date in bed_assignment
+        return Patient.objects.filter(
+            medical_records__bed_assignment__bed__room__ward=self,
+            medical_records__bed_assignment__discharge_date__isnull=True,
+        ).distinct()
```

Comment:
```suggestion
        # return Patient.objects.filter(medical_records__bed_assignment__bed__room__ward=self).distinct()
        # filter patients whos current stay is in this room, namely no discharge date in bed_assignment
        return Patient.objects.filter(
            medical_records__bed_assignment__bed__room__ward=self,
            medical_records__bed_assignment__discharge_date__isnull=True,
        ).distinct()
```
This is a good point, however I would say that we have to also consider that the discharge day may not be `null`, but in the future. I had a similar situation [here](https://github.com/charludo/ycms/blob/a4c4b3849da8e3cdd7a5aedd08e7a3a608f0fb07/ycms/cms/models/bed.py#L35), the solution would probably be very similar as well.